Library/ViewModels/PledgeAmountCellViewModel.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+import KsApi
+import Prelude
+import ReactiveExtensions
+import ReactiveSwift
+
+public protocol PledgeAmountCellViewModelInputs {
+  func configureWith(project: Project, reward: Reward)
+}
+
+public protocol PledgeAmountCellViewModelOutputs {
+  var amount: Signal<String, Never> { get }
+  var currency: Signal<String, Never> { get }
+}
+
+public protocol PledgeAmountCellViewModelType {
+  var inputs: PledgeAmountCellViewModelInputs { get }
+  var outputs: PledgeAmountCellViewModelOutputs { get }
+}
+
+public final class PledgeAmountCellViewModel: PledgeAmountCellViewModelType,
+  PledgeAmountCellViewModelInputs, PledgeAmountCellViewModelOutputs {
+  public init() {
+    let project = self.projectAndRewardProperty.signal
+      .skipNil()
+      .map(first)
+
+    let reward = self.projectAndRewardProperty.signal
+      .skipNil()
+      .map(second)
+
+    self.amount = reward
+      .map { String(format: "%.0f", $0.minimum) }
+
+    self.currency = project
+      .map { currencySymbol(forCountry: $0.country).trimmed() }
```

Comment:
Would we ever want `currencySymbol` to not be trimmed? we could move that to inside the function.

Reply:
Sure, was simply using what we used before

Reply:
I looked at the internal logic a bit more and with the internal logic it seems more appropriate to have it here (it's either using trimmed 2x at call site like we do now or adding it to possible 5 `return` statements). Of course we can add a convenience wrapper but I'm not really sure if that saves us that much since we'd have to also test it I guess.

Leaving as is unless strong opinions on this.